modular_citadel/code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -17,30 +17,3 @@
 /obj/item/gun/energy/laser/redtag
 	lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
-	
-/obj/item/gun/energy/laser/carbine
```

Comment:
by 'this already exists' I mean your addition is superfluous

Reply:
I meant to move it to non-modular files